versions.props

```diff
@@ -50,7 +50,7 @@ org.apache.kerby:*=1.0.1
 org.apache.logging.log4j:*=2.17.1
```

Comment:
This should be bumped to 2.17.2 based on `versions.lock`

Reply:
Done (not sure why it would matter though).

Reply:
It doesn't matter in this case - but since the version got bumped - been trying to keep things in `versions.props` up to whatever version is already chosen. Easier to see and not have two conflicting pieces of information when it isn't necessary.